index/models.py

```diff
@@ -119,51 +122,64 @@ def add_info_other(self, info):
 class Petition(models.Model):
     title = models.CharField(max_length=200)  # Title of petition
     description = models.CharField(max_length=4000)  # The paragraph description
-    ID = models.IntegerField(999999, primary_key=True)
+    ID = models.IntegerField(primary_key=True)
     archived = models.BooleanField(default=False)
     hidden = models.BooleanField(default=False)
 
-    # Files the date created
-    created_date = models.DateTimeField(db_index=True, default=timezone.now())
-    # The expected signature to move to the next step
-    expected_sig = models.IntegerField(300)
-    # The author of the petition
-    author = models.ForeignKey(User, on_delete=models.CASCADE, related_name="petitions")
-    # The tags, probably a max of 3
-    tags = models.ManyToManyField(Tag, related_name="petitions")
-    # The signature
-    signatures = models.ManyToManyField(Signature, related_name="petitions")
-    # If the senate has responded , their answer
+    created_date = models.DateTimeField(
+        db_index=True, default=timezone.now()
```

Comment:
default field values that are not constant should be callables instead of values: https://docs.djangoproject.com/en/2.1/ref/models/fields/#default

Reply:
This should be fixed now, thank you!